run-ovms.sh

```diff
@@ -54,35 +52,17 @@ elif grep -q "file" <<< "$INPUTSRC"; then
 	arrfilesrc=(${INPUTSRC//:/ })
 	# use vids since container maps a volume to this location based on sample-media folder
 	# TODO: need to pass demux/codec info
-	inputsrc="filesrc location=vids/"${arrfilesrc[1]}" ! qtdemux ! h264parse "
```

Comment:
so where is this logic moved to ?   i think we probably add this into gst profile

Reply:
moved to run_gst.sh